google-cloud-spanner/acceptance/spanner/client/transaction_test.rb

```diff
@@ -92,6 +94,8 @@
 
       tx.insert "accounts", additional_account
```

Comment:
No, its only there to illustrate that it won't be called (and hence won't raise an error) because the transaction has already been rolled back. I'll update it to raise an error with a more descriptive message.